cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts

```diff
@@ -40,31 +40,31 @@ describe('Project Edit, Copy, Sync', () => {
         .its('response.body')
         .then((edited: Project) => {
           expect(edited.name).to.eql(`${project.name} - edited`);
-          cy.verifyPageTitle(`${edited.name}`);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
           cy.clickButton(/^Edit project$/);
           cy.get('[data-cy="name"]').clear().type(`${project.name}`);
           cy.clickButton(/^Save project$/);
-          cy.verifyPageTitle(project.name);
+          cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
         });
     });
 
     it('can edit a project from the project details page', () => {
       cy.clickTableRowLink('name', project.name, { disableFilter: true });
-      cy.verifyPageTitle(project.name);
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
       cy.clickButton(/^Edit project$/);
-      cy.verifyPageTitle('Edit Project');
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```

Comment:
This name is not correct. As names in tests are generated it cannot be hardcoded like here. Instead something like
```
cy.verifyPageTitle(`Edit ${project.name}`);
```